<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SBXperiment</class>
 <widget class="QWidget" name="SBXperiment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,10">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max (0-10K):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,10">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max (0-10K):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox2">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="horizontalScrollBar_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
